src/v2/Apps/Components/AppShell.tsx

```diff
@@ -48,6 +49,11 @@ export const AppShell: React.FC<AppShellProps> = props => {
     document.body.setAttribute("data-test", "AppReady")
   }, [])
 
+  // Wait for route to finish rendering before (possibly) switching out the theme
```

Comment:
This is small but could we expand this comment a bit and just kinda explain whats going on here and why its needed?